jsdoc/services/jsParser.spec.js

```diff
@@ -0,0 +1,34 @@
+var Dgeni = require('dgeni');
+var mockPackage = require('../mocks/mockPackage');
+
+describe('jsParser service', function() {
+
+  var jsParser;
+
+  function testJsParserPackage() {
+    return new Dgeni.Package('testJsParserPackage', [mockPackage()])
+    .config(function(jsParser) {
+      jsParser.config = {
+        comment: false,              // this value should NOT override the default (true)
```

Comment:
I don't quite understand the comment on this line. Should this override the default or not?

Reply:
Are you saying that even if you set the config for `comment` it will not affect the value passed to the parser?

Reply:
Yes, that was my goal. Since the current implementation of the jsdoc package requires javascript comments to be attached to the espree AST in order to work properly, I forbade the user to override such options. (However now I realize that this choice goes against the general goal to provide a fully customizable tool to the users)

Anyway, in my last commit I moved the config to the new `jsParserConfig` service.